helpers/image_helper.rb

```diff
@@ -6,11 +6,14 @@
 
 # Class for working with images after convert service
 class ImageHelper
+  # The method returns the file size taking url
+  # @param url [String] url image
+  # @return [Integer] size image
   def self.get_image_size(url)
-    Tempfile.create('images-') do |fo|
-      fo.binmode
-      fo.write(URI.parse(url).open(&:read))
-      File.size(fo.path)
+    Tempfile.create('images-') do |tmpfile|
+      binary_image = URI.open(URI.parse(url), 'r:binary').read
```

Comment:
Uri.open(url).read will not close connection

https://stackoverflow.com/a/4217806/820501